Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN addgroup -S shipitron && \
 
 RUN apk add --no-cache \
     ca-certificates \
-    openssl \
+    openssl-dev \
```

Comment:
Why `dev`?

Reply:
`rugged` compiles a vendored version of `libgit2` and needs the dev package